Stripe/PublicHeaders/STPBlocks.h

```diff
@@ -106,6 +107,14 @@ typedef void (^STPSourceCompletionBlock)(STPSource * __nullable source, NSError
  */
 typedef void (^STPSourceProtocolCompletionBlock)(id<STPSourceProtocol> __nullable source, NSError * __nullable error);
 
+/**
+ A callback to be run with a PaymentIntent response from the Stripe API.
+
+ @param paymentIntent The Stripe PaymentIntent from the response. Will be nil if an error occurs. @see STPPaymentIntent
```

Comment:
do we need to specifically say `STPPaymentIntent.h` here like you did for StripeError.h? I'm just imagining this from a very unaware perspective.